[Radar] Update changelog, glossary and overview page (#20821)
cloudflare/cloudflare-docs: src/content/docs/radar/index.mdx

---

import {
	CardGrid,
	Description,
	Feature,
	LinkButton,
	LinkTitleCard,
	Plan,
} from "~/components";

<Description>

> 	Get access to Cloudflare's data on global Internet traffic.

Reviewer: ```suggestion
Get access to Cloudflare's data on global Internet traffic.
```

</Description>

<Plan type="all"/>

> [Cloudflare Radar](https://radar.cloudflare.com) is a hub that showcases global Internet traffic, attacks, and technology trends and insights. It is powered by data from [Cloudflare’s global network](https://www.cloudflare.com/network/), as well as aggregated and [anonymized](/1.1.1.1/privacy/public-dns-resolver/) data from Cloudflare’s [1.1.1.1 public DNS resolver](/1.1.1.1/).

Author: ?

> [Cloudflare Radar](https://radar.cloudflare.com) is a hub that showcases global Internet traffic, attacks, and technology trends and insights. It is powered by data from [Cloudflare’s global network](https://www.cloudflare.com/network/), as well as aggregated and [anonymized](/1.1.1.1/privacy/public-dns-resolver/) data from Cloudflare’s [1.1.1.1 public DNS resolver](/1.1.1.1/).

Reviewer: Hey @bllchmbrs ! I think someone has already mentioned this to you, but we get a lot of these where Hyperlint says "cloudflare" should be capitalized when it's in code or a URL (when it can stay lowercase). But even so, I also don't understand what changes Hyperlint is suggesting here. I'll resolve/ignore this, but just wanted to let you know. Thanks!
(cc: @kodster28 )

> [Cloudflare Radar](https://radar.cloudflare.com) is a hub that showcases global Internet traffic, attacks, and technology trends and insights. It is powered by data from [Cloudflare’s global network](https://www.cloudflare.com/network/), as well as aggregated and [anonymized](/1.1.1.1/privacy/public-dns-resolver/) data from Cloudflare’s [1.1.1.1 public DNS resolver](/1.1.1.1/).

Reviewer: ```suggestion
[Cloudflare Radar](https://radar.cloudflare.com) is a hub that showcases global Internet traffic, attacks, and technology trends and insights. It is powered by data from [Cloudflare’s global network](https://www.cloudflare.com/network/), as well as aggregated and [anonymized data](/1.1.1.1/privacy/public-dns-resolver/) from Cloudflare’s [1.1.1.1 public DNS resolver](/1.1.1.1/).
```

Using [Radar's API](/api/resources/radar/) you can access Cloudflare's data on global Internet traffic.

> Radar's API is free, allowing academics, data sleuths, and other web enthusiasts to investigate Internet usage across the globe.

Reviewer: ```suggestion
Radar's API is free, allowing academics, technology professionals, and other web enthusiasts to investigate Internet usage across the globe.
```
I don't have a super strong opinion on this. I just thought "data sleuths" may sound a bit too casual and nefarious. But if this was approved language somewhere else, feel free to ignore me.

Data available via Radar API endpoints is made available under the [CC BY-NC 4.0](https://creativecommons.org/licenses/by-nc/4.0/) license.

<LinkButton
	variant="primary"
	href="/radar/get-started/"
>
	Get started
</LinkButton>

<LinkButton
	variant="secondary"
	href="https://radar.cloudflare.com/"
>

> 	Radar website

Reviewer: ```suggestion
	Radar dashboard
```
Not sure if this is right. Just a suggestion.

Author: We prefer "website" when referring to Radar, but thanks for the suggestion!

</LinkButton>

---

## Features

<Feature
	header="Make your first API request"
	href="/radar/get-started/first-request/"
	cta="Make your first API request"
>
	Start learning how to use Radar's API by making your first request.
</Feature>

<Feature
	header="Compare data"
	href="/radar/get-started/making-comparisons/"
	cta="Compare data"
>
	What to know before making comparisons between locations, [autonomous

> 	systems](https://www.cloudflare.com/en-gb/learning/network-layer/what-is-an-autonomous-system/) and more.

Reviewer: ```suggestion
	systems](https://www.cloudflare.com/en-gb/learning/network-layer/what-is-an-autonomous-system/), and more.
```
